### PR TITLE
ING-708: Improve remaining ensure helper tests to use dinocluster

### PR DESCRIPTION
**Pull request by chvck**

### CODE REVIEW

File: cbmgmtx/ensuremanifesthelper_test.go [hunk 1]
```diff
@@ -1,79 +1,139 @@
...
-		targets = append(targets, NodeTarget{
-			Endpoint: fmt.Sprintf("http://%s:%d", nodeExt.Hostname, nodeExt.Services.Mgmt),
-			Username: testutils.TestOpts.Username,
-			Password: testutils.TestOpts.Password,
-		})
-	}
+	blockHost, execEndpoint, baseTargets := testutils.SelectNodeToBlock(t, testutils.TestServiceNs)
+	targets := testNodesToNodeTargets(t, baseTargets)
 
 	// we intentionally use the last target that will be polled as the node
```
Comment (Westwooo) [inline cbmgmtx/ensuremanifesthelper_test.go:30]: nit: don't need this comment anymore

File: cbmgmtx/mgmt_test.go [hunk 2]
```diff
@@ -191,89 +193,89 @@ func TestHttpMgmtBuckets(t *testing.T) {
...
-		isFirstError := sErr.Argument == "fieldOne"
-		if isFirstError {
-			assert.Equal(t, sErr.Reason, "reasonOne, something else")
-		} else {
-			assert.Equal(t, sErr.Argument, "fieldTwo")
-			assert.Equal(t, sErr.Reason, "reason, something")
-		}
-	})
-}
+// func Test_parseForInvalidArg(t *testing.T) {
```
Comment (Westwooo) [inline cbmgmtx/mgmt_test.go:237]: Is this supposed to be commented out/still here?

File: cbqueryx/ensureindexhelper_test.go [hunk 3]
```diff
@@ -1,79 +1,122 @@
...
+	blockHost, execEndpoint, baseTargets := testutils.SelectNodeToBlock(t, testutils.TestServiceQuery)
+	targets := testNodesToNodeTargets(t, baseTargets)
+
+	if blockHost == "" {
+		// This test requires a very specific cluster setup containing a node with the query service but not indexing.
+		// The test blocks traffic to that query node.
+		t.Skipf("Cluster not in configuration that can be used by this test")
 	}
 
 	// we intentionally use the last target that will be polled as the node
```
Comment (Westwooo) [inline cbqueryx/ensureindexhelper_test.go:48]: Same nit

File: cbqueryx/preparedquery_test.go [hunk 4]
```diff
@@ -203,5 +205,7 @@ func TestPreparedQueryPreparedNameMissing(t *testing.T) {
 
 	assertQueryResult(t, expectedRows, &expectedResult, res)
 
-	require.Empty(t, cache.queryCache)
+	stmt, ok := cache.Get("SELECT 1=1")
```
Comment (Westwooo) [inline cbqueryx/preparedquery_test.go:208]: What's this change doing?

**Review by Westwooo — DISMISSED** [review]

File: cbqueryx/preparedquery_test.go (hunk 4 above)
Comment (chvck) [reply cbqueryx/preparedquery_test.go:208]: queryCache is no longer accessible. This is looking up the statement in the cache to ensure it isn't in there.

File: cbqueryx/ensureindexhelper_test.go (hunk 3 above)
Comment (chvck) [reply cbqueryx/ensureindexhelper_test.go:48]: This is a required comment, it explains why we skip this test.

Comment (chvck) [reply cbqueryx/ensureindexhelper_test.go:48]: Nevermind...

**Comment by brett19** [comment]
Closed in favor of https://github.com/couchbase/gocbcorex/pull/256 which uses some of the most recent dino-testing changes made to gocbcorex integration testing.

**Pull request closed without merging**